hooks/helpers/logging.ps1

```
@@ -0,0 +1,49 @@
+    }
+    Write-Verbose $logMessage
+}
+
+function Handle-Error {
+    param(
+        [System.Exception]$Exception,
+        [string]$Message = ""
+    )
+
+    $errorMessage = if ($Message) { $Message } else { $Exception.Message }
+    $stackTrace = $Exception.ScriptStackTrace
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'stackTrace' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)